src/Sarif.Multitool/FileWorkItemsOptions.cs

```diff
@@ -9,10 +10,40 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
     public class FileWorkItemsOptions
     {
         [Option(
-            'f',
+            't',
             "filing-target-type",
             HelpText = "The type of system to which work items are to be filed. Must be one of 'github' and 'azureDevOps'.",
             Required = true)]
-        public string TargetType { get; internal set; }
+        public string FilingTargetType { get; internal set; }
+
+        [Option(
+            'p',
+            "project-uri",
+            HelpText = "The absolute URI of the project under which the work items are to be filed.",
+            Required = true)]
+        public string ProjectUriString { get; internal set; }
+
+        // For an explanation of the difference between ProjectUriString and ProjectUri, see FileWorkItemsCommand.ValidateOptions.
+        public Uri ProjectUri { get; internal set; }
+
+        [Option(
+            'f',
+            "filtering-strategy",
+            HelpText = "Specifies the strategy for selecting which results to file. Must be one of 'new' and 'all'.",
+            Required = true)]
+        public string FilteringStrategy { get; internal set; }
```

Comment:
>FilteringStrategy [](start = 22, length = 17)

does it make sense have all these properties (filterstrategy, groupingStrategy, filing-target-type) as enums instead of strings?

Reply:
It might. I will look into that.

---
In reply to: [306567622](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306567622) [](ancestors = 306567622)

Reply:
Yes, these s/be enums if we retain them. This provides for automatic command-line validation provided by our OSS helper (which isn't always super readable, but hey, it's free).

---
In reply to: [306569347](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306569347) [](ancestors = 306569347,306567622)

Reply:
IIRC the price is that these parameter values are case-sensitive. I will look into it.

---
In reply to: [306574126](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306574126) [](ancestors = 306574126,306569347,306567622)

Reply:
See what I did in validating the `ProjectUri` option. I found CommandLine's built-in validation inadequate and worked around it.

---
In reply to: [306577599](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306577599) [](ancestors = 306577599,306574126,306569347,306567622)

Reply:
Resolving duplicate of another comment that also asked for the enums (which I am deferring in the interest of getting bug filing working this week).

---
In reply to: [306577730](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306577730) [](ancestors = 306577730,306577599,306574126,306569347,306567622)